source/grid/grid_tools.cc

```diff
@@ -1614,8 +1614,6 @@ namespace GridTools
     return active_halo_layer;
   }
 
-
```

Comment:
we generally keep spaces between definitions of functions, so no need to remove spaces here.